apps/web/pages/booking/[uid].tsx

```diff
@@ -109,35 +120,30 @@ export default function Success(props: SuccessProps) {
     seatReferenceUid,
   } = querySchema.parse(routerQuery);
 
-  const attendeeTimeZone = props?.bookingInfo?.attendees.find(
-    (attendee) => attendee.email === email
-  )?.timeZone;
+  const attendeeTimeZone = bookingInfo?.attendees.find((attendee) => attendee.email === email)?.timeZone;
 
   const tz = props.tz ? props.tz : isSuccessBookingPage && attendeeTimeZone ? attendeeTimeZone : timeZone();
 
-  const location = props.bookingInfo.location as ReturnType<typeof getEventLocationValue>;
+  const location = bookingInfo.location as ReturnType<typeof getEventLocationValue>;
   let rescheduleLocation: string | undefined;
   if (
-    typeof props.bookingInfo.responses.location === "object" &&
-    "optionValue" in props.bookingInfo.responses.location
+    typeof bookingInfo.responses?.location === "object" &&
+    "optionValue" in bookingInfo.responses.location
   ) {
-    rescheduleLocation = props.bookingInfo.responses.location.optionValue;
+    rescheduleLocation = bookingInfo.responses.location.optionValue;
   }
 
   const locationVideoCallUrl: string | undefined = bookingMetadataSchema.parse(
-    props?.bookingInfo?.metadata || {}
+    bookingInfo?.metadata || {}
   )?.videoCallUrl;
 
-  const status = props.bookingInfo?.status;
-  const reschedule = props.bookingInfo.status === BookingStatus.ACCEPTED;
-  const cancellationReason = props.bookingInfo.cancellationReason || props.bookingInfo.rejectionReason;
+  const status = bookingInfo?.status;
+  const reschedule = bookingInfo.status === BookingStatus.ACCEPTED;
+  const cancellationReason = bookingInfo.cancellationReason || bookingInfo.rejectionReason;
 
-  const attendeeName =
-    typeof props?.bookingInfo?.attendees?.[0]?.name === "string"
-      ? props?.bookingInfo?.attendees?.[0]?.name
-      : "Nameless";
+  const attendeeName = parseName(bookingInfo.responses.name);
```

Comment:
I love setting the scheduler's name dynamically on the booking page. Although not every event type will have that field in the booking title. Could we move this into the `getEventName` and only parse for the name when `{SCHEDULER}` is a part of the booking title?

Reply:
Hey @joeauyeung, I am afraid I don't understand you completely.

Reply:
When setting up event types, you can have the name of the event be the `attendeeName`. I think we should only parse for the attendee name if the `{SCHEDULER}` place holder is a part of the event name.
![CleanShot 2024-01-12 at 13 27 28](https://github.com/calcom/cal.com/assets/65426560/db620d0f-67f3-4754-b041-2e39c395b67e)

In terms of the Stripe tests what error are you getting exactly? One thing that's not a part of that doc is locally you need to send webhook events to your local machine. You can use the Stripe CLI and use the `stripe listen --forward to localhost:3000/api/integrations/stripepayment/webhook`

Reply:
Thanks for the elaboration, I will make the necessary changes tomorrow. 

Last time I tried forwarding to `localhost:3000/api/integrations/stripepayment/webhook` but was getting an error `Not able to post` from the stripe cli. I was using the stripe cli from docker.

Will try to run it again by native package tomorrow.

Reply:
I was able to get past the `Not able to post` error by using the native stripe cli. However the test aren't passing even on `Main` branch for me. The steps & assertions in the tests seems to work as far as I understand. But end up getting an Aggregate Error. 

![image](https://github.com/calcom/cal.com/assets/86384652/448f7daf-fd5c-4e06-b45f-a6c417f00984)

Tried digging in it, but the error is bit ambiguous. On the development server output, All I get is an 
`Webhook Error: Unhandled Stripe Webhook event type { stripe event type such as account updated/ authorized }`. Which comes from [here](https://github.com/calcom/cal.com/blob/e5eb121ab75fdf0d5b5ffa1b4fef61107329e4cf/packages/features/ee/payments/api/webhook.ts#L156C1-L159C10) and is expected I believe.

Aside from all this, I see nothing suspicious that I can look into.